helpers/getUserDetails.ts

```diff
@@ -0,0 +1,18 @@
+import clientWithTokenApi from "@/apiClients/clientWithTokenApi";
+import axios from "axios";
+
+export async function getUserDetails() {
+  try {
+    const response = await clientWithTokenApi.get("/users/userDetails");
+    return response.data.data;

```

Comment:
Spróbuj uniknąć takiego zagnieżdżenia "data.data", może w odpowiedzi z Backendu przydałoby się to zmienić?